docset/winserver2022-ps/activedirectory/Get-ADComputer.md

```diff
@@ -228,17 +275,24 @@ Accept wildcard characters: False
 ```
 
 ### -Credential
-Specifies the user account credentials to use to perform this task.
-The default credentials are the credentials of the currently logged on user unless the cmdlet is run from an Active Directory module for Windows PowerShell provider drive.
-If the cmdlet is run from such a provider drive, the account associated with the drive is the default.
 
-To specify this parameter, you can type a user name, such as User1 or Domain01\User01 or you can specify a **PSCredential** object.
-If you specify a user name for this parameter, the cmdlet prompts for a password.
+Specifies the user account credentials to use to perform this task. The default
+credentials are the credentials of the currently logged on user unless the
+cmdlet is run from an Active Directory module for Windows PowerShell provider
+drive. If the cmdlet is run from such a provider drive, the account associated
+with the drive is the default.
 
-You can also create a **PSCredential** object by using a script or by using the **Get-Credential** cmdlet.
-You can then set the *Credential* parameter to the **PSCredential** object.
+To specify this parameter, you can type a user name, such as User1 or
+Domain01\User01 or you can specify a **PSCredential** object. If you specify a
+user name for this parameter, the cmdlet prompts for a password.
 
-If the acting credentials do not have directory-level permission to perform the task, Active Directory module for Windows PowerShell returns a terminating error.
+You can also create a **PSCredential** object by using a script or by using the
+**Get-Credential** cmdlet. You can then set the _Credential_ parameter to the
+**PSCredential** object.
+
+If the acting credentials do not have directory-level permission to perform the
+task, Active Directory module for Windows PowerShell returns a terminating
+error.
```

Comment:
Opportunistic reword - it's the _cmdlet_ returning an error here, not the module.

```suggestion
If the acting credentials do not have directory-level permission to perform the task, the cmdlet
returns a terminating error.
```